app/src/main/java/com/chouten/app/domain/use_case/module_use_cases/GetAllModulesUseCase.kt

```diff
@@ -0,0 +1,302 @@
+package com.chouten.app.domain.use_case.module_use_cases
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.net.Uri
+import android.provider.DocumentsContract
+import android.util.Log
+import com.chouten.app.domain.model.ModuleModel
+import com.chouten.app.domain.repository.ModuleRepository
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.withContext
+import kotlinx.serialization.json.Json
+import java.io.ByteArrayOutputStream
+import javax.inject.Inject
+
+class GetAllModulesUseCase @Inject constructor(
+    private val moduleRepository: ModuleRepository,
+    private val mContext: Context,
+    private val log: suspend (String) -> Unit
+) {
+    private val jsonParser: Json = Json { ignoreUnknownKeys = true }
+
+    private enum class ModuleSubdirectory(val print: String) {
+        HOME("Home"), INFO("Info"), MEDIA("Media"),
+    }
```

Comment:
```suggestion
    private enum class ModuleSubdirectory {
        HOME, INFO, MEDIA,
    }
```
afaik this should work the same, only thing that might fuck smth is that the resulting string is all uppercase

Reply:
Yep, that'd be the issue. I can keep the additional string but rename the value to make it more understandable why it's there?